Packs/Cyberint/Classifiers/classifier-mapper-incoming-CyberInt.json

```
@@ -1,7 +1,7 @@
 {
     "description": "",
     "feed": false,
-    "id": "abcd-5678",
+    "id": "CyberInt Incoming (mapper)",
```

```suggestion
    "id": "abcd-5678",
```
the id is not allowed be changed